app/routes/home/home.container.js

```diff
@@ -18,4 +19,4 @@ export const mapDispatchToProps = (dispatch) => bindActionCreators({
   setLanguage: LocalesActions.setLanguage,
 }, dispatch);
 
-export default connect(mapStateToProps, mapDispatchToProps)(Home);
+export default hot(module)(connect(mapStateToProps, mapDispatchToProps)(Home));
```

Comment:
So now we'll need to add this HOC to every component/container which we would like to hot reload?

Reply:
In every container is enough. We'll add it to the plop generator.